foo/foo.go

```diff
@@ -0,0 +1,3 @@
+package foo
+
+var Abc = 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var Abc should have comment or be unexported